src/components/HomeSearchBox/index.tsx

```diff
@@ -6,9 +6,29 @@ const HomeSearchBox = () => {
   const { t } = useTranslation('home')
   const router = useRouter()
   const [searchValue, setSearchValue] = useState<string>('')
+  const [active, setActive] = useState(0)
+  const [filtered, setFiltered] = useState<string[]>([])
+  const [isShow, setIsShow] = useState(false)
```

Comment:
I have 3 concerns here
1. I don't think we need `active` state.
2. Please declare all the states at the top above anything else.
3. Do you think we should name more specifically?

```suggestion
  const [searchValue, setSearchValue] = useState<string>('')
  const [filteredSuggestion, setFilteredSuggestion] = useState<string[]>([])
  const [isSuggestionShown, setIsSuggestionShown] = useState(false)
  const { t } = useTranslation('home')
  const router = useRouter()
```